CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 Unreleased changes are available as `avenga/couper:edge` container.
 
+* **Added**
```

Comment:
This is a change, not an addition, right?

Reply:
may be :)

Reply:
Done